src/mem3/src/mem3_rep.erl

```diff
@@ -426,10 +426,16 @@ find_missing_revs(Acc) ->
         #doc_info{id=Id, revs=RevInfos} = couch_doc:to_doc_info(FDI),
         {Id, [R || #rev_info{rev=R} <- RevInfos]}
     end, Infos),
-    mem3_rpc:get_missing_revs(Node, Name, IdsRevs, [
+    Missing = mem3_rpc:get_missing_revs(Node, Name, IdsRevs, [
         {io_priority, {internal_repl, Name}},
         ?ADMIN_CTX
-    ]).
+    ]),
+    lists:filter(fun({_Id, Revs, _Ancestors}) ->
+        case Revs of
+           [] -> false;
+           [_ | _] -> true
+        end
```

Comment:
```suggestion
    lists:filter(fun
        (_Id, [], _Ancestors) -> false;
        (_Id, _Revs, _Ancestors) -> true
    end, Missing). 
```

Reply:
Maybe better as separate function heads? Am not 100% and this is an awfully minor nit.